src/quo2/components/wallet/transaction_summary/view.cljs

```diff
@@ -0,0 +1,118 @@
+(ns quo2.components.wallet.transaction-summary.view
+  (:require [quo2.theme :as quo.theme]
+            [quo2.components.wallet.transaction-summary.style :as style]
+            [react-native.core :as rn]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.icon :as icon]
+            [quo2.components.tags.context-tag.view :as context-tag]
+            [utils.i18n :as i18n]))
+
+(def transaction-translation
+  {:send   [i18n/label :t/send]
+   :swap   [i18n/label :t/swap]
+   :bridge [i18n/label :t/bridge]})
```

Comment:
`i18n/label` is a function that returns a string, not hiccup, so I think we shouldn't use it as a React component